sycl/include/sycl/kernel.hpp

```diff
@@ -157,6 +158,9 @@ class __SYCL_EXPORT kernel : public detail::OwnerLessBase<kernel> {
   typename detail::is_kernel_device_specific_info_desc<Param>::return_type
       get_info(const device &Device, const range<3> &WGSize) const;
 
+  template <typename Param>
+  typename Param::return_type ext_oneapi_get_info(const queue &q) const;
+
```

Comment:
Could you also add another TODO here with a reference to https://github.com/intel/llvm/pull/7598?

I'd also like to hear from somebody else in @intel/dpcpp-specification-reviewers about this function.  Although we wrote `ext_oneapi_get_info` in the spec, I'm wondering if we should have written `ext_oneapi_experimental_get_info` (because the extension is experimental).  But I don't think I've seen that naming convention anywhere else, so I want to double check.

Reply:
We do not have a precedent for adding "experimental" to the prefix like this in any other extension.  I think we should stick with `ext_oneapi_get_info` for consistency.